README.md

```diff
@@ -78,16 +67,76 @@ pnpm install
   cd src && cp config-examples/* config && for i in config/*-example.js; do mv -- "$i" "${i%-example.js}.js"; done
   ```
 
-4. Set the app-config.js values for:
+2. Set the app-config.js values for:
 
 - `orsApiKey` - ORS API key to be used when ot running the app from localhost or ors valid domains
 - `bitlyApiKey` - the Bitly key (used to shorten the share URL)
 - `bitlyLogin` - the Bitly login (used to shorten the share URL)
 
-5. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
+3. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
 
 The filters, theme and hooks of the map client can be customized if needed.
 
+##### Run with `Docker`
+
+Run the app in `docker` in 3 steps:
+
+1. Build the image:
+
+```shell
+docker build --tag ors-map-client:local .
+```
+
+2. Run a container:
+
+```shell
+docker run -d -it -v "$(pwd)"/nginx/logs:/var/log/nginx:rw -p 127.0.0.1:8080:80 --name ors-map-client ors-map-client:local
+```
+
+3. Go to http://127.0.0.1:8080 and enjoy!
```

Comment:
Not sure about the `and enjoy!` 😅 part 
would put like:
`The app should be running at ...`

Reply:
Ah and do you think we should add commands to stop/remove the container again for `docker` and `docker compose`?
And on the same thought... is there a benefit of using the native docker command?
Or should we just remove that part and only show the docker compose, which is the most simple deployment?

Reply:
> Ah and do you think we should add commands to stop/remove the container again for `docker` and `docker compose`? And on the same thought... is there a benefit of using the native docker command? Or should we just remove that part and only show the docker compose, which is the most simple deployment?

I'd say both. Not everyone uses compose. Moreover, sometimes for a quick and dirty run, it's worth having a docker one-liner.